modules/balancer/balancer.gql

```diff
@@ -1,7 +1,41 @@
+extend type Mutation {
+    balancerMutationTest: String!
+}
+
+union SorResponse = GqlSorGetSwapsResponse | GqlCowSwapApiResponse
```

Comment:
this doesn't need to be a union, you can just return the cowswap response since this resolver is only visible in the balancer api